### PR TITLE
Material Design Icons: Add icons for new Add Manage All Mailboxes page

### PR DESCRIPTION
**Pull request by stephanethomas**
This pull request adds the following [Material Design icons](https://fonts.google.com/icons?selected=Material+Icons) to Calypso:

Icon | Name
------ | -----
 | `dvr`
 | `forward_to_inbox`
 | `mediation`
 | `move_to_inbox`
 | `smartphone`

It also refines a readme file a bit. Those changes were extracted from https://github.com/Automattic/wp-calypso/pull/52755.

### CODE REVIEW

File: packages/material-design-icons/README.md [hunk 1]
```diff
@@ -11,28 +11,28 @@ out an alternative distribution of the icons.
...
+## Adding a Material icon
 
-To add more Material Design icons, you'll have to download individual icons to appropriate directories
-then rebuild and checkin updated `material-icons.svg` file.
+To add more icons, you'll have to download individual icons to the appropriate directories,
+then rebuild and commit the updated sprites file:
 
-Select and download individual icon's SVG file from <https://material.io/tools/icons/>,
-minding the style (like `outline`) and size.
+  1. Select an icon from the [official repository](https://fonts.google.com/icons?selected=Material+Icons), minding the style (like `outline`)
```
Comment (daledupreez) [inline packages/material-design-icons/README.md]: Is the size not relevant?

File: packages/material-design-icons/README.md [hunk 2]
```diff
@@ -11,28 +11,28 @@ out an alternative distribution of the icons.
...
 
-Move the SVG files in the sub-folder matching the category used on material.io.
+  2. Download the corresponding SVG file with `black` as color
 
-Rebuild `material-icons.svg` by running:
+  3. Move that SVG file in the sub-folder matching the category of that icon
```
Comment (daledupreez) [inline packages/material-design-icons/README.md]: Does it make sense to specify that the category should be from the official repository? It's not clear where the category should come from.

**Review by daledupreez — APPROVED** [review]
I verified that the downloaded icons are the ones you identified, and they look good to me.

I do have some minor comments on the README updates, but those are not strictly blocking.

File: packages/material-design-icons/README.md (hunk 1 above)
Comment (stephanethomas) [reply packages/material-design-icons/README.md]: When searching for an icon you can't really specify a size, but only a category or a style. I will clarify that, and will also probably move up the mention of the recommended style and size (currently located at the end of that readme file).

File: packages/material-design-icons/README.md (hunk 2 above)
Comment (stephanethomas) [reply packages/material-design-icons/README.md]: I will clarify that 👍.

**Pull request merged**